docs/source/_static/data_formatting_examples/format_iss_breast_data.py

```diff
@@ -122,12 +149,18 @@ def add_codebook(experiment_json_doc):
             'nuclei': ISSCroppedBreastAuxTileFetcher(input_dir, 'nuclei'),
             'dots': ISSCroppedBreastAuxTileFetcher(input_dir, 'dots'),
         },
-        postprocess_func=add_codebook,
-        default_shape={Axes.Y: 1044, Axes.X: 1390}
     )
 
 
 if __name__ == "__main__":
+    """
```

Comment:
consider moving this to be the `description` argument passed to `argparse.ArgumentParser()`

Reply:
If we want these scripts to be user-facing and user-friendly there's a lot of work to be done across them; right now we have a mixture of click and argparse scripts. If we think it's worth cleaning these up I'd suggest we do them all in a separate PR.